Weather/ContentView.swift

```diff
@@ -8,9 +8,18 @@
 import SwiftUI
 
 struct ContentView: View {
+    @ObservedObject var cityVM = CityViewModel()
```

Comment:
It's better to use a full name: `cityViewModel`

Reply:
Fixed.)